main.go

```diff
@@ -134,10 +109,45 @@ func main() {
 	log.Info("rqlite server stopped")
 }
 
+func closeFile(f *os.File) {
+	if err := f.Close(); err != nil {
+		log.Errorf("Unable to close file: %s", err.Error())
+		os.Exit(1)
+	}
+}
+
+func createFile(path string) *os.File {
```

Comment:
See my comment below. `defer` and error checking ain't good friends, so I came up with this helper function.